napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -242,14 +243,19 @@ def changeIsoThreshold(self, value):
             Threshold for isosurface.
         """
         with self.layer.events.blocker(self._on_iso_threshold_change):
-            self.layer.iso_threshold = value / 100
+            self.layer.iso_threshold = value
+
+    def _on_contrast_limits_change(self):
+        with self.layer.events.blocker(self._on_iso_threshold_change):
+            cmin, cmax = self.layer.contrast_limits
+            self.isoThresholdSlider.setMinimum(cmin)
+            self.isoThresholdSlider.setMaximum(cmax)
+        return super()._on_contrast_limits_change()
```

Comment:
Awesome! As above, I would use `range` instead, but otherwise exactly what we need.